modules/monitoring-example-remote-write-queue-configuration.adoc

```diff
@@ -3,28 +3,37 @@
 // * observability/monitoring/configuring-the-monitoring-stack.adoc
 
 :_mod-docs-content-type: REFERENCE
-[id="example-remote-write-queue-configuration_{context}"]
-= Example remote write queue configuration
 
-// Set attributes to distinguish between cluster monitoring examples and user workload monitoring examples.
-ifndef::openshift-dedicated,openshift-rosa[]
+// The ultimate solution DOES NOT NEED separate IDs, it is just needed for now so that the tests will not break
+// tag::CPM[]
+[id="example-remote-write-queue-configuration-cpm_{context}"]
+= Example remote write queue configuration (CPM)
+// end::CPM[]
+
+// tag::UWM[]
+[id="example-remote-write-queue-configuration-uwm_{context}"]
+= Example remote write queue configuration (UWM)
```

Comment:
if user know what does UWM mean here

Reply:
Thank you for asking about this! I added the note to explain that the change is just temporary to make reviews easier. Thank you!

Reply:
https://84002--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/monitoring/configuring-the-monitoring-stack#example-remote-write-queue-configuration-uwm_configuring-the-monitoring-stack
title is `Example remote write queue configuration (UWM)`, UWM is not replaced